oemof/solph/custom.py

```diff
@@ -11,15 +11,20 @@
 SPDX-License-Identifier: GPL-3.0-or-later
 """
 
+
 from pyomo.core.base.block import SimpleBlock
 from pyomo.environ import (Binary, Set, NonNegativeReals, Var, Constraint,
-                           Expression, BuildAction)
+                           Expression, BuildAction, Reals)
 import logging
 
-from oemof.solph.network import Bus, Transformer, Flow
+from oemof.solph.network import Bus, Transformer, Flow, Sink
 from oemof.solph.plumbing import sequence
 
 
+
```

Comment:
According to PEP8, there should be exactly two blank lines before class definitions.